### ISSUE TITLE
Heap histogram interanl stats producing unique class names

### ISSUE THREAD

**Issue by at055612** [gchq/stroom#609]
The heap histogram internal stats are producing class names with anonymous inner class identifiers and lambda identifiers.   There are not particularly useful and will consume a lot of UIDs in stroom-stats.  

Suggest doing a string replace using a regex from a global prop to replace something like `$$4493dc63` with `$$--REPLACED--`.

```
stroom.query.audit.client.DocRefResourceHttpClient$$Lambda$46/1402766141
stroom.statistics.server.sql.SQLStatisticAggregationTransactionHelper$$EnhancerBySpringCGLIB$$ce71a393$$FastClassBySpringCGLIB$$4493dc63
```

**Comment by at055612** [comment]
The stats now look like this after replacement: ```

stroom.explorer.server.ExplorerTreeModel$$EnhancerBySpringCGLIB$$--ID-REMOVED--
--
stroom.explorer.server.ExplorerTreeModel$$EnhancerBySpringCGLIB$$--ID-REMOVED--$$FastClassBySpringCGLIB$$--ID-REMOVED--
stroom.explorer.server.ExplorerTreeModel$$FastClassBySpringCGLIB$$--ID-REMOVED--

```

**Comment by at055612** [comment]
The regex for replacing the ID values in the class name is now configured via 
`stroom.node.status.heapHistogram.classNameReplacementRegex`

**Comment by at055612** [comment]
Fixed in > 6.0-alpha.8
